### PR TITLE
Adds "nyaaaaaaa" to brain damage lines

### PR DESCRIPTION
**Pull request by harmonyn**
damn you antur
no fun allowed

### CODE REVIEW

**Comment by Bawhoppen** [comment]
>inb4 giant essay-sized discussion spam on this

**Comment by meeeeme** [comment]
add a ~

File: strings/brain_damage_lines.json [hunk 1]
```diff
@@ -57,7 +57,8 @@
         "i will snatch erry motherfucker birthday",
         "the ai and borgs are mettacomming I think",
         "u just did the world a little bit more sad place for someone",
-        "WHERES THE SLIP REWRITE WHERE THR FUCK ID IT?"
+        "WHERES THE SLIP REWRITE WHERE THR FUCK ID IT?",
+        "NYAAAAAA"
```
Comment (Xhuis) [inline strings/brain_damage_lines.json]: You should have the possibility to have tildes at the end, plus the occasional stuttered N.

**Comment by DoctoreMad** [comment]
For fuck sakes I can't believe the CODERBUS METACLUB is once again trying to stampede on japanese rights on this station if youve evr met a REAL JAPANESE you'd notice that his cranium can instantly compute the meaning of the universe so if he makes some shit you eat it up like you shoueld eat catgirls you fucking Skubfag should probably uncommit everythign and dump this PR and hope nobody remembrs it because if they do maybe japanese hacker will find IP and fucking CRACK you and nteorare your're waifu with his internet fingers and paid African-American bodyguard. If only you could have held back tis PR you would not be running around with your wife across the streets hopgn you don't have to prpe the bul then all would be fine but you didnt you couldn't so now youre gonna pay the price I am already calling my japanese _WAIFU_ to get your Prs deleted and U better hoppe she doeds not dislike your real life face becauase she cna order the **Yakuza** to come down ur house and btfo u(1 second all it takes!!!) delet this PR DELETE IT NOW ITS AN ORDER RREEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

**Comment by Xhuis** [comment]
You haven't even been on the GitHub for two weeks and you're already going insane.

**Comment by Xhuis** [comment]
Better.

**Comment by FantasticFwoosh** [comment]
Assuming anybody goes into the process of commenting github without being insane in the first place.

**Comment by VeryRealAccount** [comment]
It needs a tilde at the end of it

**Comment by banthebantz** [comment]
scream

**Pull request merged**